drivers/ethernet/eth_sam_gmac.c

```diff
@@ -2139,38 +2139,44 @@ static struct device DEVICE_NAME_GET(eth0_sam_gmac);
 
 static void eth0_irq_config(void)
 {
-	IRQ_CONNECT(GMAC_IRQn, CONFIG_ETH_SAM_GMAC_IRQ_PRI, queue0_isr,
```

Comment:
Convert this to use new DT_INST macro syntax.

Reply:
Done